components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -265,6 +265,50 @@ - (void)testElevationNormalZeroElevation {
   XCTAssertEqual([button elevationForState:UIControlStateNormal], 0);
 }
 
+- (void)testDefaultBorderWidth {
+  // Given
+  MDCButton *button = [[MDCButton alloc] init];
+
+  // Then
+  for (NSUInteger controlState = 0; controlState < kNumUIControlStates; ++controlState) {
```

Comment:
Looks like this isn't testing the `.selected | .highlighted | .disabled` state. Please make the comparison `<=` so it will test the combination of all UIControl states.  Same for the other 2 tests being added.

Reply:
Done